pkg/controller/cluster/controller.go

```diff
@@ -473,3 +516,52 @@ func (e *external) getPayload(ctx context.Context, ref *v1alpha1.SecretReference
 
 	return nil, nil
 }
+
+// ExtractKubeconfigFromSecretRef extracts login information from a Kubeconfig Secret Reference
+func (e *external) ExtractKubeconfigFromSecretRef(ctx context.Context, p *v1alpha1.ClusterParameters, r *argocdv1alpha1.Cluster) error {
```

Comment:
Should be lower case as its only used internally.

```suggestion
func (e *external) extractKubeconfigFromSecretRef(ctx context.Context, p *v1alpha1.ClusterParameters, r *argocdv1alpha1.Cluster) error {
```

Reply:
renamed to lowercase